jaxrs-tck/docs/assertions/JAXRSSpecAssertions_3.1.0.xml

```diff
@@ -1399,7 +1399,7 @@
         <!-- JAXRS 3.1-->
         <assertion required="true" impl-spec="false" defined-by="technology" status="active" testable="true">
             <id>JAXRS:SPEC:131</id>
-            <description>There are two alternative ways of publishing on Java SE: Creating SE endpoints directly or using the SE bootstrap API. Any compliant implementation that supports either of these alternatives on Java SE MUST support both.</description>
+            <description>There are two alternative ways of publishing on Java SE: Creating SE endpoints directly or using the SE bootstrap API. A compliant implementation MUST support both alternatives on Java SE.</description>
```

Comment:
This is not a correct change. It was changed recently in https://github.com/jakartaee/rest/pull/1231, please remove this change

Reply:
I assume the changes in PR1231 is required for release 4.0, hence added the same in new file jaxrs-tck/docs/assertions/JAXRSSpecAssertions_4.0.0.xml. Is that ok.

Reply:
Oh, missed that. [PR 1231](https://github.com/jakartaee/rest/pull/1231) is appropriate for the 3.1 release, but I seem unnecessary to apply it there now.  The associated tests were made optional a while ago under a challenge.